torch/csrc/jit/passes/lower_tuples.cpp

```diff
@@ -0,0 +1,114 @@
+#include "torch/csrc/jit/passes/lower_tuples.h"
+#include "torch/csrc/jit/passes/dead_code_elimination.h"
+#include "torch/csrc/utils/functional.h"
+
+namespace torch { namespace jit {
+
+// operators where we expect to find tuples as inputs/outputs
+// this is to assert we are only  doing modifications when we know
+// we can flatten tuples
+std::unordered_set<Symbol> white_list = {
+  prim::If,
+  prim::Loop,
+  prim::TupleUnpack,
+  prim::TupleConstruct,
+  prim::Param,
+  prim::Return,
+ };
+
+
+static void LowerTuples(Block* block);
```

Comment:
Why are these functions static?

Reply:
Aren't they internal to the pass? Only `void LowerTuples(Graph& graph)` should be public

Reply:
Is there a reason to use `static` over an anonymous namespace?

Reply:
Not really, a matter of taste I guess